ATTRIBUTIONS.md

```diff
@@ -23,3 +23,207 @@
 >   http://www.apache.org/licenses/LICENSE-2.0
 > 
 > Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+# Android Beacon Library
```

Comment:
Maybe we can link to the license like in the others, and not duplicate it whole? Not sure though